Makefile

```
@@ -697,13 +696,33 @@ deploy-docs: venv
+
+.PHONY: demo-e2e
+demo-e2e:
+	@command -v curl >/dev/null 2>&1 || { echo "Error: curl not found in PATH."; exit 1; }
+	@command -v jq >/dev/null 2>&1 || { echo "Error: jq not found in PATH."; exit 1; }
+	@mkdir -p $(DEMO_OUTPUT_DIR)
+	KUBECONFIG=$(KUBECONFIG_DIR)/operator-controller-experimental-e2e.kubeconfig \
+	DEMO_OUTPUT_DIR=$(DEMO_OUTPUT_DIR) go test -count=1 -v ./test/e2e/features_test.go -timeout 30m \
+	  -args --godog.tags="@demo" --godog.concurrency=1
+
+.PHONY: demo-svg
+demo-svg: #EXHELP Convert asciicast recordings to SVG.
```

checkmake (0.3.2)

[warning] Target body for "demo-svg" exceeds allowed length of 5 lines (7).

(maxbodylength)